docs/usage.rst

```diff
@@ -12,11 +12,12 @@ Basic Usage Examples
 
 
 The BigchainDB driver's main purpose is to connect to one or more BigchainDB
-server nodes, in order to perform supported API calls documented under
-:doc:`drivers-clients/http-client-server-api`.
+server nodes, in order to perform supported API calls (see the server's
+`HTTP API docs <https://docs.bigchaindb.com/projects/server/en/latest/drivers-clients/http-client-server-api.html>`_
```

Comment:
I think we need to handle https://github.com/bigchaindb/bigchaindb-driver/issues/105